frontend/redux/actions.js

```diff
@@ -57,6 +57,30 @@ export const requestAttributesFailed = (error, retryAction) => ({
   error: true,
 });
 
+/** @prop {string} Action to trigger a request for face data */
+export const REQUEST_OVERALL_STATS = 'REQUEST_OVERALL_STATS';
+export const requestOverallStats = () => ({
+  type: REQUEST_OVERALL_STATS,
+});
+
+/** @prop {string} Action to populate the annotations within the store when the request succeeds */
```

Comment:
Yes. I used many for a base since the new stuff is _very_ similar. Most of the others should have their proper wording.